Resources/Prototypes/Entities/Markers/Spawners/Random/posters.yml

```
@@ -169,6 +169,6 @@
       - PosterLegitShoukou # Nyanotrasen Poster, see Resources/Prototypes/Nyanotrasen/Entities/Structures/Wallmount/Signs/posters.yml
       - PosterLegitCornzza # Nyanotrasen Poster, see Resources/Prototypes/Nyanotrasen/Entities/Structures/Wallmount/Signs/posters.yml
       - PosterLegitFuckAround # DeltaV Poster, see Resources/Prototypes/DeltaV/Entities/Structures/Wallmount/Signs/posters.yml
-      - PosterLegitOppenhopper
+    # - PosterLegitOppenhopper
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)